fotoapparat/src/main/java/io/fotoapparat/hardware/v1/RawCameraParametersProvider.java

```diff
@@ -0,0 +1,82 @@
+package io.fotoapparat.hardware.v1;
+
+import android.hardware.Camera;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Provides {@link Camera.Parameters} with methods to extract additional camera parameters.
+ */
+@SuppressWarnings("deprecation")
+public class RawCameraParametersProvider {
+
+    /* Raw camera params keys */
+    private static final String[] RAW_ISO_SUPPORTED_VALUES_KEYS = {
+            "iso-values", "iso-mode-values", "iso-speed-values", "nv-picture-iso-values"
+    };
+    private static final String[] RAW_ISO_CURRENT_VALUE_KEYS = {
```

Comment:
this is actually unused. Are you sure that `applySensorSensitivity` will work across different devices?

Reply:
Hmm, will think about better organizing `applySensorSensitivity` work using `RAW_ISO_CURRENT_VALUE_KEYS`.